### PR TITLE
added: Html and styles

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**

### CODE REVIEW

**Review by vrx29 — APPROVED** [review]
Looks good you can merge this branch

**Review by vrinda-mahajan — APPROVED** [review]
The code looks clean.👍

**Comment by Dev-AkshatGupta** [comment]
> The code looks clean.👍

Thanks for reviewing man

**Comment by Dev-AkshatGupta** [comment]
> Thanks for reviewing man



> Looks good you can merge this branch

Thanks for reviewing man

File: index.html [hunk 1]
```diff
@@ -12,11 +12,131 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (parvez-rashid) [inline index.html:136]: Whitespace can be removed

File: documentation.css [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    margin-bottom: 2rem;
+    background-color: var(--DEE2E6);
+    font-weight: 800;
+    width: inherit;
+}
+
+/*----------------- navigation menu */
+
+.component-navigation {
+
```
Comment (parvez-rashid) [inline documentation.css:56]: Whitespace can be removed

**Review by parvez-rashid — COMMENTED** [review]
There are extra whitespaces (iI've not commented on all of them), removing them will make code look consistent.

**Comment by Megha-Joshi** [comment]
Looks Good

File: documentation.css (hunk 2 above)
Comment (Dev-AkshatGupta) [reply documentation.css:56]: I will do that now afterwards thanks for noticing Man and Thanks for review

**Pull request merged**